### PR TITLE
Add schema description to `src/mixs/schema/mixs.yaml`

### PR DESCRIPTION
**Pull request by sujaypatil96**
Notes about the text for the description from Pier can be found here: https://docs.google.com/document/d/1HCmKUGFBm4P8dPCXqL_jOuXe02SF5jQ8hPnTlhanm1Q/edit?usp=sharing

### CODE REVIEW

**Comment by sujaypatil96** [comment]
Feel free to suggest changes to the description.

**Comment by sujaypatil96** [comment]
@pbuttigieg wanted to get your thoughts on this description for the schema. Do let me know if you want me to modify it somehow.

File: src/mixs/schema/mixs.yaml [hunk 1]
```diff
@@ -1,4 +1,11 @@
 name: mixs
+description: >-
+  The MIxS (Minimum Information about any (x) Sequence) standard is specified in a YAML file
```
Comment (pbuttigieg) [inline src/mixs/schema/mixs.yaml:3]:
```suggestion
  This file contains a YAML-formatted specification of the Minimum Information about any (x) Sequence (MIxS) standard, generated using LinkML (https://linkml.io/linkml/). This file is released by the Genomic Standards Consortium (GSC; https://www.gensc.org/) for use by anyone handling data or information about biological sequences. This file is also used as an authoritative 'source of truth' to generate downstream GSC artifacts, available here: https://github.com/GenomicsStandardsConsortium/mixs/tree/main/project
```

File: src/mixs/schema/mixs.yaml [hunk 2]
```diff
@@ -1,4 +1,11 @@
 name: mixs
+description: >-
+  The MIxS (Minimum Information about any (x) Sequence) standard is specified in a YAML file 
+  using [LinkML](https://linkml.io/linkml/). This standard is released by the [GSC](https://www.gensc.org/) 
+  (Genomic Standards Consortium) for use by anyone handling data or information about biological sequences. 
+  The YAML file can be found [here](https://github.com/GenomicsStandardsConsortium/mixs/tree/main/src/mixs/schema), 
+  and the downstream artifacts generated from the YAML schema can be found 
+  [here](https://github.com/GenomicsStandardsConsortium/mixs/tree/main/project).
```
Comment (pbuttigieg) [inline src/mixs/schema/mixs.yaml:8]:
```suggestion
```

**Review by pbuttigieg — CHANGES_REQUESTED** [review]
There should be no line breaks or Md formatting in the yaml description field. 

I also made the language a little more precise.

I removed the link to the file itself (that was a self-reference right?) - that should be in a dedicated field

**Review by pbuttigieg — APPROVED** [review]
Green light from me

**Pull request merged**